### ISSUE TITLE
Add option for set app theme (light or cinema mode) depending of the system device

### ISSUE THREAD

**Issue by sguinetti** [realOxy/M3UAndroid#50]
I just saw the cinema mode feature (OLED Theme) and I like it. But I would prefer it to be activated according to the system settings, especially when the system has programmed the dark theme to match this feature (with following automatically the scheduled dark mode feature of the system). I appreciate this consideration in advance.

Example:
Cinema mode option in the app:
* Yes (everyday/always)
* [Follow] System (e.g. only nights in most devices)
* No (never)

See this for more context: https://developer.android.com/develop/ui/views/theming/darktheme#change-themes

* Text has edited to avoid misunderstandings.

**Comment by realOxy** [comment]
Starting from the complete migration to material3, the cinema mode has become to always use the dark theme, and its description in English and Simplified Chinese has changed. 😹

**Comment by realOxy** [comment]
Because the OLEN theme color is not easy to configure in material3, I simply redid this feature.

**Comment by sguinetti** [comment]
I understand. But, i'm not talking about changing the color or anything like that. I happened to think it was nice that the cinema mode is automatically activated at sunset or after thanks to the Android system. In the phone, I have the option enabled to automatically switch from light to dark at night.

To give you an idea, cell phones include the option to do it automatically: https://android.gadgethacks.com/how-to/make-android-10s-dark-mode-turn-automatically-sunset-0267092/

**Comment by realOxy** [comment]
The effect of the current cinema mode is: always use the dark theme, that is, it does not follow the system theme changes. There are no plans to mark the app-level feature of switching themes based on sunset time at the moment.
In addition, I will launch a custom theme system in subsequent versions.

**Comment by sguinetti** [comment]
I understand. Then I will close this issue for now. Sorry if I misunderstood, I had to explain better, because what I would propose is to follow the dark theme as it is in the system settings. However, after many proposals in this issue, I welcome the option of a "custom theme system" for this app. Several preinstalled apps of recent Android smatphones already include that option and that's why I created this issue.
